pubspec.yaml

```diff
@@ -1,14 +1,17 @@
 name: amazon_cognito_identity_dart_2
-version: 0.1.24
+version: 0.1.24+1
 homepage: https://github.com/furaiev/amazon-cognito-identity-dart-2
-description: Unofficial Amazon Cognito Identity Provider Dart SDK,
-  to add user sign-up / sign-in to your mobile and web apps
-  with AWS Cloud Services. Based on amazon-cognito-identity-dart
+description: Unofficial Amazon Cognito Identity Provider Dart SDK, to add user
+  sign-up / sign-in to your mobile and web apps with AWS Cloud Services. Based
+  on amazon-cognito-identity-dart
 
 dependencies:
+  flutter:
+    sdk: flutter
+  convert: ^2.1.1
   crypto: ^2.0.2+1
   http: ^0.12.1
-  convert: ^2.1.1
+  shared_preferences: ^0.5.12+4
```

Comment:
@bsneider thank you for the contribution, it seems like a typo - `shared_preferences` should be updated in `example/pubspec.yaml`, not here.

Could you please also add a line to CHANGELOG? Appreciate your work.

Reply:
I thought that originally too, but then I figured if we update here to an re-release the dependency override is no longer needed, so I did it in both places

Reply:
@furaiev do you have an example CHANGELOG.md format you're looking for?

Reply:
https://github.com/furaiev/amazon-cognito-identity-dart-2/blob/master/CHANGELOG.md

Reply:
I believe it's a bad idea to add one more dependency. An app that uses this package can not use shared_preferences but it still will be in the dependencies. So, let's remove it from the package `pubspec.yaml`.

Reply:
sounds good removing it now and updating the changelog

Reply:
should the package version increase in the main pubspec.yaml?  0.1.24+1  or 0.1.25, or no changes bc the main code was not changed? Not sure how pub.dev gets updated if version does to change.

Reply:
0.1.24+1 will be good, thanks